src/guiloop/x11.rs

```diff
@@ -440,6 +440,11 @@ impl GuiEventLoop {
     fn process_xcb_event(&self, event: &xcb::GenericEvent) -> Result<(), Error> {
         if let Some(window_id) = Self::window_id_from_event(event) {
             self.process_window_event(window_id, event)?;
+        } else { // [FIXME] can't identify window from xkb keyboard event
```

Comment:
travis will complain about this not being rust formatted; make sure that you `cargo fmt` so that you get a green build status